test/sagemaker_tests/pytorch/inference/integration/local/test_serving.py

```diff
@@ -50,6 +53,24 @@
 }
 
 
+def disable_token_auth(framework_version):
```

Comment:
nit: could you rename this function to something along the line of "set_disable_token_auth_env"? That way it is more clear about what it does and how we use it.

Reply:
Updated, that's much more clear.